### PR TITLE
Add Elapsed Time to Timer

### PR DESCRIPTION
**Pull request by claire50**
Fixes Issue #12

### CODE REVIEW

File: TimerView.cpp [hunk 1]
```diff
@@ -5,7 +5,8 @@
 TimerView::TimerView()
 	:
 	BTextView("timer"),
-	fRunning(false)
+	fRunning(false),
+	elapsedTime(0)
```
Comment (owenca) [inline TimerView.cpp]:
```suggestion
	fElapsedTime(0)
```

Comment (owenca) [reply TimerView.cpp]: From [Haiku Coding Guidelines](https://www.haiku-os.org/development/coding-guidelines/):
identifiers

File: TimerView.cpp [hunk 2]
```diff
@@ -30,16 +31,17 @@ bool TimerView::Running()
 void TimerView::Stop()
 {
 	fRunning = false;
+	elapsedTime = Elapsed();
```
Comment (owenca) [inline TimerView.cpp]:
```suggestion
	fElapsedTime = Elapsed();
```

File: TimerView.cpp [hunk 3]
```diff
@@ -30,16 +31,17 @@ bool TimerView::Running()
...
 int TimerView::Elapsed()
 {
-	return fRunning ? (system_time() - fStartTime) / 1000 / 1000 : 0;
+	return (system_time() - fStartTime) / 1000 / 1000;
 }
 
 void TimerView::Pulse()
 {
-	bigtime_t elapsed = Elapsed();
+	bigtime_t elapsed = fRunning ? Elapsed() : elapsedTime;
```
Comment (owenca) [inline TimerView.cpp]:
```suggestion
	bigtime_t elapsed = fRunning ? Elapsed() : fElapsedTime;
```

File: TimerView.h [hunk 4]
```diff
@@ -19,6 +19,7 @@ class TimerView : public BTextView {
 private:
 	bool fRunning;
 	bigtime_t fStartTime;
+	int elapsedTime;
```
Comment (owenca) [inline TimerView.h]:
```suggestion
	int fElapsedTime;
```

**Review by owenca — CHANGES_REQUESTED** [review]
Works for me! Just need minor changes for Haiku's naming conventions.

**Comment by claire50** [comment]
Hi! I have changed the code a little bit, since I found a simpler way to fix the issue. I was having problems with committing onto GitHub, but I recloned it and changed the code again, so now it works. Thanks!

**Comment by owenca** [comment]
Thanks for making the fix even better!

**Pull request merged**